client/src/app/app.component.ts

```diff
@@ -1,46 +1,37 @@
 import { Component, OnInit, HostBinding } from '@angular/core';
 import { Router, NavigationStart, NavigationEnd, NavigationError, NavigationCancel } from '@angular/router';
-import { SlimLoadingBarService } from 'ng2-slim-loading-bar';
+import { LoadingBarService } from './shared/loading-bar/loading-bar.service';
 
 declare const require: any;
 const { version } = require('../../package.json');
 
 @Component({
   selector: 'ml-app',
   template: `
-    <ng2-slim-loading-bar color="#FFC107" height="3px"></ng2-slim-loading-bar>
+    <ml-loading-bar></ml-loading-bar>
     <router-outlet></router-outlet>
   `,
   styles: [
     `
     :host {
       display: block;
     }
-
-    ng2-slim-loading-bar {
-      height: 3px;
-      position: absolute;
-      top: 0;
-      left: 0;
-      right: 0;
-      z-index: 6;
-    }
   `
   ]
 })
 export class AppComponent implements OnInit {
   @HostBinding('attr.ml-version') version = version;
 
-  constructor(private router: Router, private slimLoadingBarService: SlimLoadingBarService) {}
+  constructor(private router: Router, private loadingBarService: LoadingBarService) {}
 
   ngOnInit() {
     this.router.events.subscribe(event => {
       if (event instanceof NavigationStart) {
-        this.slimLoadingBarService.start();
+        this.loadingBarService.start();
       } else if (event instanceof NavigationEnd) {
-        this.slimLoadingBarService.complete();
+        this.loadingBarService.stop();
       } else if (event instanceof NavigationError || event instanceof NavigationCancel) {
-        this.slimLoadingBarService.reset();
+        this.loadingBarService.stop();
```

Comment:
Omitting `reset()` and `complete()` would kind of be a UI regression (considering that we used the "determinate" style of progressbar before), but I'm fine with that for now. 

However, we might want to reintroduce completion and reset in the future.